### DISCUSSION TITLE
Falcosc / enable-loudness-equalisation

### DISCUSSION THREAD

**Discussion by kenu23** [Falcosc/enable-loudness-equalisation#26] [Q&A]
Dear Community

I would be very happy if someone could help me!

When I enter the code in PowerShell, something happens, but the enhancement tab is still missing. Do I need to enter a code in CMD before PowerShell, or is there another step I'm missing? I'm trying to set up the enhancement tab for a Virtual Hi-Fi Cable, which worked effortlessly for my friends, but for some reason, the enhancement tab doesn't show up for me.

Answer: It sounds like you're experiencing an issue with setting up the enhancement tab for your Virtual Hi-Fi Cable in PowerShell. Firstly, there shouldn't be a need to enter a code in CMD before PowerShell unless there are specific dependencies or configurations required for your setup.

To troubleshoot, ensure that the code you're entering in PowerShell is correct and properly executed. Additionally, check if there are any specific instructions provided by the manufacturer or online resources for setting up the enhancement tab. Sometimes, there might be additional software or drivers needed for certain features to work properly.

If you're still facing difficulties, consider reaching out to the manufacturer's support or community forums for assistance. They might have insights or solutions tailored to your specific setup or any known issues with the enhancement tab.

Im new here and dos´nt know anything about coding if it is easy i can do it myself but i would be ready to pay falcosc if he can add me the enhancement tab.

Thank you for answearing

**Comment by Falcosc** [comment]
First check if the script was executed completely and if yes, then you already explored everything which is possible to enable the microsoft version of loudness equaisation.

The next most easy step would be to buy a Soundcard and return it when it doesn't work out of the box or without the script, since no money could add the missing driver support if the script wasn't unable to expose it. Soundcards are dirt cheap, so cheap that they even get integrated in USB headsets without noticeable price increase compared to analog versions of the same product.

**Reply by kenu23** [reply]
Hey thank you for the fast replay.

Still have 2 questions if i buy a sound card that would change the possibility to add sound enhancement for a virtual Hi-Fi cabel and if so can you recommend me one? And all i have to to is to past in your code in powershell to turn it on right?

**Reply by Falcosc** [reply]
It needs to go through the sound card without virtual stuff.
You can't alter how the virtual Hi-Fi cable software is working. You can only avoid it by connecting your Sound output to a sound card which ships with supported drivers.

Or look outside of Microsoft Loudness Equalization for other software solutions. But I don't have any experience with other solutions, I just wrote the script for my HDMI output. Other software audio processing solutions will work with any driver, even with your virtual Hi-Fi cable

**Reply by kenu23** [reply]
Hey thank you for helping me,

But nothing worked out for me maybe because i dont understand a lot of coding! I wont get it, at least 10 friends of me did it without any problems.
